rust/src/shards/gui/mod.rs

```diff
@@ -66,6 +66,9 @@ impl EguiId {
     }
   }
 }
+pub struct Context<'a> {
```

Comment:
Missing new line

```suggestion

pub struct Context<'a> {
```

In addition, does it need to be publicly available from outside (e.g. third-party code)?
I would also rename it to `InternalContext` or `EguiContextWrapper` directly here, to not be confused with all the other context types we have 😅.